@ecomplus/storefront-template/template/js/lib/load-widgets.js

```diff
@@ -100,6 +100,10 @@ Promise.all(widgetsLoadPromises).then(() => {
         '@ecomplus/widget-ebit',
         () => import('@ecomplus/widget-ebit')
       )
+      loadWidget(
+        '@ecomplus/widget-offers-notification',
+        () => import('@ecomplus/widget-offers-notification')
+      )
```

Comment:
Acho que podemos importar esse cara só na página de produtos, certo @talissonf ?
```suggestion
      if (resource === 'products') {
        loadWidget(
          '@ecomplus/widget-offers-notification',
          () => import('@ecomplus/widget-offers-notification')
        )
      }
```

Reply:
Minha sugestão é meio que um mock, mas melhor que importar em outras views desnecessariamente, na home principalmente...